client/src/styles/Navbar.css

```diff
@@ -0,0 +1,31 @@
+.logoImg {
+  background-image: url('../../public/assets/logo.png');
+  width: 45px;
+  height: 45px;
+}
+
+/* .workoutImage {
+  background-image: url('../../public/assets/background.png');
+  height: 100vh;
+} */
+
+.fitbuddy-name {
+  padding-left: 60px;
+  font-size: 30px;
+  color: whitesmoke;
+}
+
+.navbar-morphism {
+  background-color: rgba(255, 255, 255, 0.3);
+  position: fixed;
```

Comment:
My recommendation, is to change position to unset.

Reply:
got it